### PR TITLE
Initial commit to get basic presubmit checks

### PR DESCRIPTION
**Pull request by sebastienvas**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    bazel.setVars()
+  }
+  // PR on master branch
+  if (utils.runStage('PRESUBMIT')) {
+    presubmit(gitUtils, bazel, utils)
+  }
+}
+
+def presubmit(gitUtils, bazel, utils) {
+  goBuildNode(gitUtils, 'istio.io/pilot') {
```
Comment (ayj) [inline Jenkinsfile]: s/pilot/galley ?

Comment (sebastienvas) [inline Jenkinsfile]: Done

**Comment by ayj** [comment]
@geeknoid, @rshriram - One of you will need to merge these PRs. galley-hackers doesn't seem to have write permissions.

**Comment by geeknoid** [comment]
I fixed the repo permissions, you should be able to push now.

**Pull request merged**